2024/10/25 13:33:46 [error] 35616#36156: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:33:46 [error] 35616#36156: *7 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:33:48 [error] 35616#36156: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:3000/favicon.ico", host: "localhost", referrer: "https://localhost/"
2024/10/25 13:33:48 [error] 35616#36156: *7 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:3000/favicon.ico", host: "localhost", referrer: "https://localhost/"
2024/10/25 13:33:58 [error] 35616#36156: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:33:58 [error] 35616#36156: *7 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:34:48 [error] 35616#36156: *7 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:34:48 [error] 35616#36156: *7 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:35:01 [error] 39376#39996: *17 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:35:01 [error] 39376#39996: *17 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:35:08 [error] 39376#39996: *17 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:35:08 [error] 39376#39996: *17 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:38:39 [error] 25160#25988: *24 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:38:39 [error] 25160#25988: *24 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:39:12 [error] 25160#25988: *24 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:39:12 [error] 25160#25988: *24 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "localhost"
2024/10/25 13:39:18 [error] 25160#25988: *24 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /docs HTTP/1.1", upstream: "http://127.0.0.1:3000/docs", host: "localhost"
2024/10/25 13:39:18 [error] 25160#25988: *24 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /docs HTTP/1.1", upstream: "http://127.0.0.1:3000/docs", host: "localhost"
2024/10/25 15:31:44 [error] 25160#25988: *35 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /docs HTTP/1.1", upstream: "http://127.0.0.1:3000/docs", host: "localhost"
2024/10/25 15:31:44 [error] 25160#25988: *35 CreateFile() "C:/nginx/html/50x.html" failed (3: The system cannot find the path specified), client: 127.0.0.1, server: localhost, request: "GET /docs HTTP/1.1", upstream: "http://127.0.0.1:3000/docs", host: "localhost"
2024/10/25 17:21:35 [error] 27112#44320: *76 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:3000/", host: "localhost"
2024/10/25 17:23:50 [error] 27112#44320: *85 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /static/js/bundle.js.map HTTP/1.1", upstream: "http://[::1]:3000/static/js/bundle.js.map", host: "localhost"
2024/10/25 17:33:18 [error] 38316#27472: *94 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "http://[::1]:3000/", host: "localhost"
2024/10/25 17:35:09 [error] 38316#27472: *102 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /static/js/bundle.js HTTP/1.1", upstream: "http://[::1]:3000/static/js/bundle.js", host: "localhost", referrer: "https://localhost/"
2024/10/25 17:54:54 [error] 40004#32312: *128 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.219.126, server: localhost, request: "GET / HTTP/1.1", upstream: "http://121.0.0.1:3000/", host: "192.168.219.161"
2024/10/25 18:10:15 [error] 39084#38612: *148 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.219.126, server: localhost, request: "GET / HTTP/1.1", upstream: "http://121.0.0.1:3000/", host: "192.168.219.161"
2024/10/25 18:10:36 [error] 39084#38612: *148 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.219.126, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "http://121.0.0.1:3000/favicon.ico", host: "192.168.219.161", referrer: "https://192.168.219.161/"
2024/10/25 19:53:09 [error] 39084#38612: *162 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 19:53:21 [error] 39084#38612: *164 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 19:53:34 [error] 39084#38612: *162 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 19:53:39 [error] 39084#38612: *168 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:00:30 [error] 40620#29012: *172 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:01:04 [error] 40620#29012: *172 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:01:11 [error] 40620#29012: *174 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:01:17 [error] 40620#29012: *178 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:01:18 [error] 40620#29012: *181 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:01:19 [error] 40620#29012: *184 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:01:20 [error] 40620#29012: *187 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:01:24 [error] 40620#29012: *190 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:01:32 [error] 40620#29012: *174 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:03:18 [error] 40620#29012: *197 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:03:21 [error] 40620#29012: *199 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://localhost:3000/"
2024/10/25 20:04:07 [error] 40620#29012: *203 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:05:46 [error] 40620#29012: *208 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:05:49 [error] 40620#29012: *210 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:06:55 [error] 40620#29012: *210 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:07:35 [error] 40620#29012: *210 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:07:38 [error] 40620#29012: *215 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:07:57 [error] 40620#29012: *210 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:08:08 [error] 40620#29012: *215 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:08:40 [error] 40620#29012: *215 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:09:21 [error] 40620#29012: *210 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:11:49 [error] 40620#29012: *223 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:16:59 [error] 40620#29012: *228 client intended to send too large body: 2691476 bytes, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:17:29 [error] 40620#29012: *229 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:17:33 [error] 40620#29012: *232 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:17:34 [error] 40620#29012: *235 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:17:36 [error] 40620#29012: *238 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:18:01 [error] 40620#29012: *238 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:20:36 [error] 40620#29012: *243 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:23:22 [error] 40620#29012: *248 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:23:40 [error] 40620#29012: *250 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:27:01 [error] 40620#29012: *254 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:27:49 [error] 40620#29012: *254 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:29:20 [error] 40620#29012: *260 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/25 20:31:09 [error] 40620#29012: *265 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "POST /api/ask HTTP/1.1", upstream: "http://121.0.0.1:8000/api/ask", host: "localhost", referrer: "http://127.0.0.1:3000/"
2024/10/28 18:18:31 [error] 43672#26492: *285 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.219.129, server: localhost, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "192.168.219.161"
2024/10/28 18:22:21 [error] 43672#26492: *288 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.219.129, server: localhost, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "192.168.219.161"
2024/10/28 19:05:53 [error] 25372#6364: *310 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.219.161, server: localhost, request: "GET / HTTP/1.1", upstream: "https://192.168.219.1:3000/", host: "192.168.219.161"
2024/10/28 19:06:15 [error] 25372#6364: *310 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.219.161, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "https://192.168.219.1:3000/favicon.ico", host: "192.168.219.161", referrer: "https://192.168.219.161/"
2024/10/28 19:25:47 [error] 36936#29052: *315 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 192.168.219.129, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://192.168.219.1:3000/", host: "192.168.219.161"
2024/10/28 20:43:23 [error] 37620#44316: *339 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 192.168.219.129, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "192.168.219.161"
2024/10/28 20:44:11 [error] 37620#44316: *342 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 192.168.219.129, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "192.168.219.161"
2024/10/28 20:44:14 [error] 37620#44316: *345 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 192.168.219.129, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "192.168.219.161"
2024/10/28 20:49:21 [error] 37620#44316: *348 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 192.168.219.129, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "192.168.219.161"
2024/10/28 20:52:19 [error] 37620#44316: *353 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 192.168.219.161, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "192.168.219.161"
2024/10/28 20:52:19 [error] 37620#44316: *353 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 192.168.219.161, server: 192.168.219.1, request: "GET /favicon.ico HTTP/1.1", upstream: "https://127.0.0.1:3000/favicon.ico", host: "192.168.219.161", referrer: "https://192.168.219.161/"
2024/10/28 20:54:36 [error] 37620#44316: *360 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 192.168.219.129, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "192.168.219.161", referrer: ""
2024/10/28 20:54:55 [error] 37620#44316: *362 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 192.168.219.129, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "192.168.219.161"
2024/10/29 15:03:06 [error] 9452#32588: *365 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 127.0.0.1, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "be06-220-123-122-32.ngrok-free.app"
2024/10/29 15:03:35 [error] 9452#32588: *367 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 127.0.0.1, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "be06-220-123-122-32.ngrok-free.app"
2024/10/29 15:03:39 [error] 9452#32588: *367 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 127.0.0.1, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "be06-220-123-122-32.ngrok-free.app"
2024/10/29 15:03:54 [error] 9452#32588: *370 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 127.0.0.1, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "be06-220-123-122-32.ngrok-free.app"
2024/10/29 15:29:34 [error] 42624#50304: *372 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 127.0.0.1, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://[::1]:3000/", host: "be06-220-123-122-32.ngrok-free.app"
2024/10/29 15:29:34 [error] 42624#50304: *372 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 127.0.0.1, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "be06-220-123-122-32.ngrok-free.app"
2024/10/29 16:01:34 [error] 8852#38952: *377 SSL_do_handshake() failed (SSL: error:1408F10B:SSL routines:ssl3_get_record:wrong version number) while SSL handshaking to upstream, client: 127.0.0.1, server: 192.168.219.1, request: "GET / HTTP/1.1", upstream: "https://127.0.0.1:3000/", host: "be06-220-123-122-32.ngrok-free.app"
2024/10/29 18:36:28 [error] 14848#42084: *665 connect() failed (10061: No connection could be made because the target machine actively refused it) while connecting to upstream, client: 192.168.219.120, server: f1cd-220-123-122-32.ngrok-free.app, request: "POST /api/ask?timestamp=1730194586835 HTTP/1.1", upstream: "http://127.0.0.1:8000/api/ask?timestamp=1730194586835", host: "vision21.synology.me", referrer: "https://vision21.synology.me/"
2024/10/29 20:08:37 [error] 14848#42084: *814 upstream prematurely closed connection while reading upstream, client: 192.168.219.129, server: f1cd-220-123-122-32.ngrok-free.app, request: "POST /api/ask?timestamp=1730200107057 HTTP/1.1", upstream: "http://127.0.0.1:8000/api/ask?timestamp=1730200107057", host: "vision21.synology.me", referrer: "https://vision21.synology.me/"
